cmd/captplanet/setup.go

```diff
@@ -54,8 +58,15 @@ func cmdSetup(cmd *cobra.Command, args []string) (err error) {
 	if err != nil {
 		return err
 	}
-	identPath := filepath.Join(hcPath, "ident")
-	_, err = peertls.NewTLSFileOptions(identPath, identPath, true, true)
+	// TODO: handle setting base path *and* identity file paths via args
+	// NB: if base path is set this overrides identity and CA path options
+	if setupCfg.BasePath != defaultConfDir {
```

Comment:
if setupCfg.BasePath == defaultConf, setupCfg.HCCA.CertPath and setupCfg.GWCA.CertPath are the same thing, which is bad. i think for setup it really will be simpler to just not worry about letting people configure these separately

Reply:
see comment below about farmer ca setup